src/main/config/messages_template.xml

```diff
@@ -7,413 +7,415 @@ Copyright 2005, 2006 IBM Corporation
+    <reason>The '%2' plug-in cannot be loaded because it requires at least one plug-in in '%1'.</reason>
+    <response>Make sure all prerequisite plug-ins are properly installed.</response>
   </message>
-  
+
+  <message id="DOTJ021E" type="ERROR">
+    <reason>No output generated for '%1' because all content has been filtered out by DITAVAL 'exclude' conditions, or the resource is not valid DITA.</reason>
+    <response>Check the '%1' resource and the DITAVAL file to see if this is the intended result.</response>
+  </message>
+
   <!-- Message DOTJ021W deprecated in 3.4, replaced with DOTJ021E -->
   <message id="DOTJ021W" type="WARN">
-    <reason>File '%1' will not generate output since it is invalid or all of its content has been filtered out by the ditaval file.</reason>
-    <response>Please check the file '%1' and the ditaval file to see if this is the intended result.</response>
+    <reason>No output generated for '%1' because all content has been filtered out by DITAVAL 'exclude' conditions, or the resource is not valid DITA.</reason>
+    <response>Check the '%1' resource and the DITAVAL file to see if this is the intended result.</response>
   </message>
-  
-  <message id="DOTJ021E" type="ERROR">
-    <reason>File '%1' will not generate output because because all content has been filtered out by DITAVAL "exclude" conditions, or because the file is not valid DITA.</reason>
-    <response></response>
-  </message>
-  
+
   <message id="DOTJ022F" type="FATAL">
-    <reason>Failed to parse the input file '%1' because all of its content has been filtered out.</reason>
-    <response>This will happen if the input file has filter conditions on the root element, and a ditaval excludes all content based on those conditions.</response>
+    <reason>Failed to parse the input resource '%1' because all of its content has been filtered out.</reason>
+    <response>This can happen if the resource has filter conditions on the root element, and a DITAVAL file excludes all content based on those conditions.</response>
   </message>
 
   <message id="DOTJ023E" type="ERROR">
-    <reason>Failed to get the specified image file '%1', so it will not be included with your output.</reason>
-    <response></response>
+    <reason>The specified image file '%1' is not available. It is not included in the output.</reason>
+    <response>Make sure '%1' exists and that you have permission to access it.</response>
   </message>
-  
+
   <message id="DOTJ025E" type="ERROR">
-    <reason>The input to the "topic merge" transform process could not be found.</reason>
-    <response>Correct any earlier transform errors and try the build again, or see the DITA-OT documentation for additional causes.</response>
+    <reason>The input to the 'topic merge' process cannot be found.</reason>
+    <response>Correct any earlier errors and try the build again, or see the documentation for additional causes.</response>
```

Comment:
For this and the next one, I don't see any docs specifically about topic merge failures; we might just want to remove that comment, or if the content exists, be more explicit about what to search for.

Reply:
🐛 Looks like Algolia may not be indexing table cells. 

The docs reference there is to the **Additional details** column that is pushed to the message listing topic:

- https://www.dita-ot.org/dev/topics/error-messages#msgs__DOTJ025E
- https://www.dita-ot.org/dev/topics/error-messages#msgs__DOTJ026E

The content there comes from the docs repo [topics/error-messages-details.xml#L176-L195](https://github.com/dita-ot/docs/blob/develop/topics/error-messages-details.xml#L176-L195).
We can always tweak that as necessary.